hardkernel/odroid_h4/README.md

```diff
@@ -0,0 +1,26 @@
+# Blobs
+
+The directory contains blobs to produce a full Dasharo firmware image for
+Hardkernel ODROID H4+ platforms.
+
+Do not use or load software from this repository until you have carefully
+read the licenses assigned to the relevant components. By downloading or
+using the software component from this repository, you agree to the terms
+of license associated with the given software component. If you do not wish
+to so agree, do not install or use the software from this repository.
+Association of the software components and licenses is presented below.
+
+- me.bin - Intel Management Engine
+    Version: v16.50.10.1351,
+    License: PV Intel OBL Software License Agreement 11.2.2017
```

Comment:
You didn't copy the template I have provided, did you?

In my template license was a link. Please fix.

Reply:
I did not. I have used a README for another platform instead, naively thinking that it's the same as what you wrote in the documentation, judging by a quick look.

https://github.com/Dasharo/dasharo-blobs/pull/23/commits/176bc1cb26cdefd1908c90fe6c5217fab58620a2#diff-708d08b771dcd3dbf87889870402c75472255110ace4191ddb8fb65e4f143feaR13-R18